extras/mmu/mmu_selector.py

```diff
@@ -1584,6 +1584,25 @@ def restore_gate(self, gate):
         else:
             self.servo_state = self.mmu.FILAMENT_UNKNOWN_STATE
 
+    def filament_drive(self):
+        if self.mmu.gate_selected >= 0:
+            angle = self.selector_angles[self.mmu.gate_selected]
+            self.mmu.log_trace("Setting servo to filament grip position at angle: %.1f" % angle)
+            self._set_servo_angle(angle)
+            self.servo_state = self.mmu.FILAMENT_DRIVE_STATE
+
+    def filament_release(self, measure=False):
+        if self.mmu.gate_selected >= 0:
+            release_angle = self._get_closest_released_angle()
+            self.mmu.log_trace("Setting servo to filament released position at angle: %.1f" % release_angle)
+            self._set_servo_angle(release_angle)
+            self.servo_state = self.mmu.FILAMENT_RELEASE_STATE
+        return 0. # Encoder movement
+
+    def filament_hold(self):
+        # TODO: same as release?
+        self.filament_release()
```

Comment:
I'm not really sure what the intent with "hold" is. Looking at the LinearSelector, it looked like hold also more/less raises the selector similar to filament release. I'm sure there's more nuances to it though. Should this do something other than do filament release?